.circleci/config.yml

```diff
@@ -261,11 +268,23 @@ jobs:
 
 workflows:
   version: 2
+  nightly:
+    triggers:
+      - schedule:
+          cron: "0 0 * * *"
+          filters:
+            branches:
+              only: master
+    jobs:
+      - x86-64-unknown-linux-gnu-nightly
+
   tests:
     jobs:
       # sanity tests
       - verify-changelog
       - validate-shell-scripts
+      # temp
+      - x86-64-unknown-linux-gnu-nightly
```

Comment:
This "on every CI" is temporary and will be removed once I verify is it working.